collections.md

```diff
@@ -1725,7 +1721,7 @@ The `reduceWithKeys` method reduces an associative collection to a single value,
 
     $collection->reduceWithKeys(function ($carry, $value, $key) use ($ratio) {
         return $carry + ($value * $ratio[$key]);
-    }, 0);
+    });
```

Comment:
I previously added 0 as `$initialValue` to the example to make it clear that `reduceWithKeys` also accepts initial value. But now that it's merged into `reduce` and we already show that in the previous example there is no need for that and we can remove it to simplify and make it more readable.